src/webassembly/instance.rs

```diff
@@ -85,6 +85,12 @@ impl fmt::Debug for EmscriptenData {
     }
 }
 
+#[derive(PartialEq)]
+pub enum InstanceABI {
```

Comment:
Should we also make the `Instance` struct support more abi types?

Reply:
I think we can add new abi types as we keep implementing them.
Thoughts?

Reply:
Like how wil we store different abi datas inside the instance stuct? Since it currently has a field called emscripten data. Would we just add more fields?

Reply:
Oh! I like that idea!
Would you like to handle that in a separate PR?